Makefile.kokkos

```diff
@@ -1264,7 +1316,9 @@ ifeq ($(KOKKOS_INTERNAL_USE_SYCL), 1)
 endif
 
 ifeq ($(KOKKOS_INTERNAL_DISABLE_DESUL_ATOMICS), 0)
-  tmp := $(call kokkos_append_header,"$H""define KOKKOS_ENABLE_IMPL_DESUL_ATOMICS")
+  ifeq ($(KOKKOS_INTERNAL_USE_OPENACC), 0)
+    tmp := $(call kokkos_append_header,"$H""define KOKKOS_ENABLE_IMPL_DESUL_ATOMICS")
+  endif
 else
```

Comment:
I would consider forcing `KOKKOS_INTERNAL_DISABLE_DESUL_ATOMICS` to be `1` for `OpenACC` and leave this alone.